ui/src/pages/Create.js

```diff
@@ -0,0 +1,645 @@
+          </option>
+          <option value='other'>
+            {this.props.intl.formatMessage(messages.sexOther)}
+          </option>
+        </Select>
+        <TextField
+          label={this.props.intl.formatMessage(messages.age)}
+          outlined
+        >
+          <Input
+            name='age'
+            value={this.state.formAge}
+            onChange={(e) => this.setState({formAge: e.target.value})} />
+        </TextField>
+        <TextField
+          label={this.props.intl.formatMessage(messages.homeStreetAddress)}
+          outlined
+        >
+          <Input
+            name='home_street'
+            value={this.state.formHomeStreetAddress}
+            onChange={(e) => this.setState({
+                formHomeStreetAddress: e.target.value,
+            })} />
+        </TextField>
+        <TextField
+          label={this.props.intl.formatMessage(messages.city)}
+          outlined
+        >
+          <Input
+            name='home_city'
+            value={this.state.formCity}
+            onChange={(e) => this.setState({formCity: e.target.value})} />
+        </TextField>
+        <TextField
+          label={this.props.intl.formatMessage(messages.provinceOrState)}
+          outlined
+        >
+          <Input
+            name='home_state'
+            value={this.state.formProvinceState}
+            onChange={
+              (e) => this.setState({formProvinceState: e.target.value})} />
+        </TextField>
+        <TextField
+          label={this.props.intl.formatMessage(messages.country)}
+          outlined
+        >
+          <Input
+            name='home_country'
+            value={this.state.formCountry}
+            onChange={(e) => this.setState({formCountry: e.target.value})} />
+        </TextField>
+        {/*
+        TODO(nworden): add description field. We may need to implement our
+        own notched textarea field due to an issue with the Material
+        component for it:
+        https://github.com/material-components/material-components-web-react/issues/559
+        */}
+        {this.renderPhotoFields()}
+        {this.renderProfilePageFields()}
+      </div>
+    );
+  }
+
+  renderIdentifyingInfoFields() {
+    var moreFields = this.state.showAllIdInfoFields ?
+        this.renderMoreIdentifyingInfoFields() :
+        this.renderMoreFieldsButton();
+    return (
+      <div className='create-formsectionwrapper'>
+        <div className='create-formgroupwrapper'>
+          <span className='mdc-typography--overline'>
+            Identifying information
+          </span>
+          <TextField
+            label={this.props.intl.formatMessage(
+                messages.familyOrSurameRequired)}
+            outlined
+          >
+            <Input
+              name='family_name'
+              value={this.state.formSurname}
+              onChange={(e) => this.setState({formSurname: e.target.value})} />
+          </TextField>
+          <TextField
```

Comment:
Since there are so many of them with the same structure, you can factor it out as a method (or even a component):
```js
this.renderTextField(
  'formGivenName',
  'given_name',
  messages.givenNameOrFirstNameRequired)
```